test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -978,31 +972,68 @@
                              :producer_timestamp test-time})
       (scf-store/deactivate-node! "foo4"))))
 
-(def factset-results
-  (map #(utils/assoc-when % "timestamp" reference-time "producer_timestamp" reference-time)
-       [{"facts" {"domain" "testing.com"
-                  "uptime_seconds" "4000"
-                  "test#~delimiter" "foo"
-                  "my_structured_fact" {"d" {"n" ""}, "e" "1", "c" ["a" "b" "c"]
-                                        "f" nil, "b" 3.14, "a" 1}},
+;; FACTSETS TRANSFORMATION
+
+(defn strip-expanded
+  "Strips out expanded data from the query results if the database is HSQLDB"
+  [{:strs [facts] :as record}]
+  (if (sutils/postgres?)
+    (update-in record ["facts" "data"] set)
+    (assoc record "facts" (dissoc facts "data"))))
+
+(defn munge-actual-factset
+  [factset]
+  (if (sutils/postgres?)
+    (update-in factset ["facts" "data"] set)
+    factset))
+
+(defn munge-actual-factsets
```

Comment:
"actual" is a bit confusing (but I get it). Maybe munge-factset-response or munge-factset-test-response?